effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -212,7 +227,7 @@ class RecursiveDescentParsers(positions: Positions, tokens: Seq[Token]) {
     }
 
   /*
-  <tryExpr> ::= try <stmts> <handler>+
+  <tryExpr> ::= try { <stmts> } <handler>+
```

Comment:
The braces are actually not necessary. It should be Stmt which can be braced statements

Reply:
Yes, you're right